src/content/getusermedia/gum/js/test.js

```diff
@@ -5,98 +5,80 @@
  *  that can be found in the LICENSE file in the root of the source
  *  tree.
  */
- /* eslint-env node */
-
+/* eslint-env node */
 'use strict';
+
 // This is a basic test file for use with testling.
 // The test script language comes from tape.
-var test = require('tape');
+const test = require('tape');
+
+const webdriver = require('selenium-webdriver');
+const seleniumHelpers = require('webrtc-utilities').seleniumLib;
 
-var webdriver = require('selenium-webdriver');
-var seleniumHelpers = require('webrtc-utilities').seleniumLib;
+const path = '/src/content/getusermedia/gum/index.html';
+const url = (process.env.BASEURL ? process.env.BASEURL : ('file://' + process.cwd())) + path;
 
-test('Video width and video height are set on GUM sample', function(t) {
+test('Video width and video height are set on GUM sample', t => {
```

Comment:
Does test() or equivalent allow for "async t = { ... }" instead of "t => { ... }"?

Reply:
Yes. I plan to migrate to async/await in a later PR. Didn't want to make too much changes in this PR.

Reply:
Ok sgtm!